web-sdk/v3.md

```diff
@@ -151,7 +161,27 @@ const label = {
   <tr>
     <td><code>callback</code></td>
     <td>function</td>
-    <td>Function to be called when the form is submitted.</td>
+    <td><b>Deprecated</b> Function to be called when the card enroll succeeds or fails.</td>
```

Comment:
I think that since the alternative callbacks are just below this one the deprecation callout is enough. If the documentation was scattered I would consider adding links for the other callbacks (similar to your suggestion).